hack/gen-repository-iso/dockerfile.centos7

```diff
@@ -9,7 +9,7 @@ RUN mkdir ${DIR} && cd ${DIR} && \
     yum-config-manager --add-repo https://download.docker.com/linux/centos/docker-ce.repo && \
     yumdownloader --resolve --destdir ./ docker-ce-${DOCKERVERSION} docker-ce-cli-${DOCKERVERSION} containerd.io \
         nss nss-sysinit nss-tools lz4 chrony bind-utils nfs-utils net-tools ipvsadm ceph-common glusterfs-fuse yum-utils \
-        curl openssl socat conntrack ebtables ipset telnet lvm2 keepalived haproxy psmisc tcpdump && \
+        curl openssl socat conntrack ebtables ipset ipvsadm telnet lvm2 keepalived haproxy psmisc tcpdump && \
```

Comment:
`ipvsadm` is already included in these three dockerfile. 😉

Reply:
removed the redundant parts...